comp/core/workloadmeta/def/types.go

```diff
@@ -1553,9 +1553,18 @@ type Service struct {
 	// TracerMetadata contains APM tracer metadata
 	TracerMetadata []tracermetadata.TracerMetadata
 
-	// DDService is the value from DD_SERVICE environment variable
+	// DDService is a service name currently based on either the DD_SERVICE
+	// environment variable, the DD_TAGS environment variable, or other
+	// parsing of framework-specific tracer configuration of service names, such
+	// as Java tracer properties on the command line.
+	//
+	// The UST.Service field on the other hand contains the raw value of the
+	// DD_SERVICE environment variable.
 	DDService string
 
+	// UST contains Unified Service Tagging environment variables
+	UST UST
```

Comment:
Should this be added to the protobufs?
The process entity was added recently in https://github.com/DataDog/datadog-agent/pull/40522